src/python/foglamp_daemon.py

```diff
@@ -21,9 +23,7 @@ def do_something(logf):
     logger.addHandler(fh)
     logger.setLevel(logging.DEBUG)
 
-    # set DB config
-    Configurator().initialize_dbconfig()
-    CoAPServer.start()
+    start.start()
```

Comment:
we should have classes as needed. and definitely can have more than 1 class to better make them organized